### ISSUE TITLE
Check what the SOPs in the emobon biosamples includes 

### ISSUE THREAD

**Issue by kmexter** [emo-bon/sequencing-data#16]
Following my extensive analysis of ILVOs BioSample metadata I should do the same on our emo bon ones
In particular 
- do we use URLs whereever we can
- how do we refer to SOPs
- Is it clear from the metadata fields what gene type (if just one) is included
- do we include genoscope SOPs in the experiement metadata (if that is what the experiment metadata are supposed to include)?
In other words, to a complete outsider, do our ENA metadata provide FULL provenance and reusability?

**Comment by kmexter** [comment]
Pinging @cpavloud just as an FYI - I will pass this by you if I have any recommendations to make

**Comment by kmexter** [comment]
Note to self: 
For the samples that are going for shotgun metagenomics, we will have one run accession per sample accession, unless sequencing is repeated. In that case, we will have multiple run accessions for one sample accession.

For the samples that are going for metabarcoding, we will have (at least) two run accessions (one per gene) for one sample accession.

**Comment by cpavloud** [comment]
> In other words, to a complete outsider, do our ENA metadata provide FULL provenance and reusability?

I cannot think of any study that provides more provenance than us. 
But in order to be sure about it, we will need to have outside eyes to look at it. 

This is an example of what is visible in ENA regarding protocols

![Image](https://github.com/user-attachments/assets/ff3f572a-0c50-492a-9bc2-81475f5e486a)


The only thing that is missing is the doi for the Handbook. 
But we need to add the 2024 version (which includes all the DNA related protocols) in OBPS first. 
I am tagging @isanti here, to keep an eye on it.

**Comment by kmexter** [comment]
looking at the screenshot:
- indeed the sample material processing is better given as a URL, as the others are. however, to do that we need to either have the individual SOPs taken out and "published" separated, or have the URL end in #WaSOP1. I will ask @marc-portier about this second possibility
- relvant SOP - while I dont understand the words there, can these be a URLs or is that not necessary (As in, will an omic person know exactly what is meant here?) 

On other fields visible in the screenshot
 geographic location cannot be a lat, long?
- CTD-system is good, but I wonder if we can move to using vocabulary for this - maybe we need to work with BODC in having an emo bon space in there for all our instruments, as SDN etc do? Some of our instruments are in BODC already, but not all of them. WDYT, @isanti, could you handle the communication to BODC to ask about this?
- emobon as project name fine, but can we have a URL as well?

**Comment by cpavloud** [comment]
> On other fields visible in the screenshot
geographic location cannot be a lat, long?

There are also coordinates in decimals, this is just a screenshot of some of the fields. 


> emobon as project name fine, but can we have a URL as well?

Not really, I don't think ENA would allow it. 


> relvant SOP - while I dont understand the words there, can these be a URLs or is that not necessary (As in, will an omic person know exactly what is meant here?)

This is related to the library preparation, i.e. the molecular part of the work.

**Comment by kmexter** [comment]
Yes, but are they socuments or titles of methods? If docs, can they be provided as url?

**Comment by cpavloud** [comment]
> Yes, but are they socuments or titles of methods? If docs, can they be provided as url?

I have to check.

**Comment by kmexter** [comment]
After the bebopping, we should go back and change all SOP for samples to be the URL, meaning that when we bebop them, we need to do that also for the first version of any protocol - so publish on zenodo first the first one and then the second ones, etc
